cognite_toolkit/examples/cdf_oid_example_data/raw/files_metadata.csv

```diff
@@ -0,0 +1,12 @@
+name,source,mime_type,doc_type
```

Comment:
because this is closer to what we have in a project setting - the files will not come out of the source system contextualized, with an external ID and mapped to a data set.  So more natural to add some processing to add this...

Reply:
I think maybe I didn't quite understand you on slack wrt this specific change. How do you upload them to files?
I thought you would upload the files bare minimum (i.e. with external_id = file name, and then contextualise them afterwards.

Reply:
as I wrote above - trying to make it look a bit closer to what a project will have.